test/dlc_tests/conftest.py

```diff
@@ -413,6 +414,16 @@ def gpu_only():
     pass
 
 
+@pytest.fixture(scope="session")
+def x86_compatible_only():
+    pass
+
+
+@pytest.fixture(scope="session")
+def graviton_only():
```

Comment:
nit: do we want have it the same naming pattern as x86, like `graviton_compatible_only`